galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/RunQuery.java

```diff
@@ -67,7 +68,16 @@ public class RunQuery extends HttpServlet {
 
 	@Override
 	protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+		try{
+			retrieveResults(req, resp);
+		}catch (Exception e){
+			String msg = new ServletError(GAL5000_GENERIC_API_ERROR).toString();
+					sendResponse(resp, msg, HttpServletResponse.SC_NOT_FOUND);
+					logger.error(msg,e);
+		}
+	};
 
+	protected void retrieveResults(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
 		int pageNum = 1;
 		int pageSize = 100;
```

Comment:
I know this isn't part of your PR - but why define these values here when we give them new values in 2 lines time?  Can we clean this up a bit?

Reply:
Fixed in latest code.